core/css.js

```diff
@@ -172,7 +172,7 @@ Blockly.Css.CONTENT = [
     'display: none;',
     'font-family: sans-serif;',
     'font-size: 9pt;',
-    'opacity: 0.9;',
+    'opacity: .9;',
```

Comment:
Google's CSS style guide says no leading zeros in CSS (but yes to leading zeros in JS).